crates/ggml/sys/build.rs

```diff
@@ -332,6 +332,7 @@ fn enable_cublas(build: &mut cc::Build, out_dir: &Path) {
             .arg("static")
             .arg("--generate-code=arch=compute_52,code=[compute_52,sm_52]")
             .arg("--generate-code=arch=compute_61,code=[compute_61,sm_61]")
+            .arg("--generate-code=arch=compute_75,code=[compute_75,sm_75]")
```

Comment:
Is this needed for newer cards?

Reply:
Yes  I had issues with my card RTX2070, I guess we should also support 8.x architectures like A100